### PR TITLE
[JENKINS-72148] Add missing fr properties in hudson directory

### PR DESCRIPTION
**Pull request by jgreffe**
See [JENKINS-72148](https://issues.jenkins.io/browse/JENKINS-72148).
Add and translate all the missing `*_fr.properties` in src/main/resources/hudon and src/main/resources/lib.
Missing `*_fr.properties` within src/main/resources/jenkins will be processed in another PR.

### Proposed changelog entries

- JENKINS-72148, add missing fr properties in hudson and lib resources

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

File: core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/description_fr.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+blurb = Informe les administrateurs à propos de l''osbolescence d''un ou plusieurs plugins installés. \
+  Les sites de mise à jour fournissent ces données, la documentation du plugin indique généralement pourquoi le plugin est obsolète.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/description_fr.properties:2]:
```suggestion
  Cette information est fournise par les sites de mise à jour, la documentation du plugin indique généralement pourquoi le plugin est obsolète.
```
More close to the original sentence, subjective & optional.

File: core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/message_fr.properties [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+
+DeprecatedPlugins=Les plugins suivants sont obsolètes
+DeprecatedPluginsDescription=\
+  En général, cela signifie que ces plugins sont soit obsolètes, ne sont plus dévelopés, ou ne fonctionnent plus.<br />\
+  Voir les pages liées pour plus d''informations sur la cause de l''obsolescence, et des suggestions sur la procédure.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/message_fr.properties:27]:
```suggestion
  Voir les pages liées pour plus d''informations sur la cause de cette obsolescence, et des suggestions sur la procédure à suivre.
```
Optional

File: core/src/main/resources/hudson/cli/Messages_fr.properties [hunk 3]
```diff
@@ -0,0 +1,101 @@
+InstallPluginCommand.DidYouMean={0} semble être un nom court de plugin. Vouliez-vous ‘{1}’?
+InstallPluginCommand.InstallingFromUpdateCenter=Installation de {0} depuis le centre de mise à jour
+InstallPluginCommand.InstallingPluginFromLocalFile=Installation d''un plugin depuis un fichier local: {0}
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/cli/Messages_fr.properties:3]:
```suggestion
InstallPluginCommand.InstallingPluginFromLocalFile=Installation d''un plugin depuis un fichier local : {0}
```

File: core/src/main/resources/hudson/cli/Messages_fr.properties [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+InstallPluginCommand.NoUpdateDataRetrieved=Aucune donnée n''a été récupérée du centre de mise à jour : {0}
+InstallPluginCommand.NotAValidSourceName={0} n''est pas un fichier valide, ni une URL, ni un nom d''artefact de plugin dans le centre de mise à jour
+
+EnablePluginCommand.NoSuchPlugin=Aucun plugin trouvé avec le nom {0}
+EnablePluginCommand.MissingDependencies=Impossible d''activer le plugin {0} car il manque la dépendance {1}
+
+AddJobToViewCommand.ShortDescription=\
+ Ajoute des jobs à la vue.
+BuildCommand.ShortDescription=\
+ Construit un job, et optionnellement attend sa complétion.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/cli/Messages_fr.properties:16]:
```suggestion
 Construit un job, et attend optionnellement sa complétion.
```
Nit, subjective

File: core/src/main/resources/hudson/cli/Messages_fr.properties [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+DeleteViewCommand.ShortDescription=\
+ Supprime une/des vue(s).
+DeleteJobCommand.ShortDescription=\
+ Supprime un/des job(s).
+EnablePluginCommand.ShortDescription=\
+ Active de façon transitive un ou plusieurs plugins installés.
+GroovyCommand.ShortDescription=\
+ Exécute le script Groovy spécifié. 
+GroovyshCommand.ShortDescription=\
+ Exécute un terminal interfactif groovy.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/cli/Messages_fr.properties:36]:
```suggestion
 Exécute un terminal interactif groovy.
```
Typo

File: core/src/main/resources/hudson/cli/Messages_fr.properties [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+HelpCommand.ShortDescription=\
+ Liste toutes les commandes disponibles ou la description détaillée d''une commande.
+InstallPluginCommand.ShortDescription=\
+ Installe un plugin soit à partir d''un fichier, une URL ou depuis le centre de mise à jour.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/cli/Messages_fr.properties:40]:
```suggestion
 Installe un plugin soit à partir d''un fichier, d''une URL ou depuis le centre de mise à jour. 
```

File: core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/message_fr.properties [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# The MIT License
+# 
+# Copyright (c) 2004-2012, Dominik Bartholdi
```
[the remark this answers is not in the document]
Comment (jgreffe) [reply core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/message_fr.properties:3]: @jtnord 
It's actually a copy of the related https://github.com/jenkinsci/jenkins/blob/64e0d59f8d2f8c32fe5e8eba519e5e6cd765a4f2/core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/message.properties.
I can remove all headers of added files `.*_fr.properties` if needed.

File: core/src/main/resources/hudson/cli/Messages_fr.properties [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+ReloadJobCommand.ShortDescription=Recharge un/des job(s)
+OnlineNodeCommand.ShortDescription=Reprise avec un nœud dédié aux builds, afin d''annuler la commande "offline-node" précédente.
+ClearQueueCommand.ShortDescription=Efface la file d''attente de build.
+ReloadConfigurationCommand.ShortDescription=Efface toutes les données chargées en mémoire et recharge l''intégralité depuis les fichiers système. Utile quand vous avez modifié des fichiers de configuration directement sur disque.
+ConnectNodeCommand.ShortDescription=Se reconnecte à un nœud.
+DisconnectNodeCommand.ShortDescription=Se déconnecte d''un nœud.
+QuietDownCommand.ShortDescription=Stoppe Jenkins silencieusement en préparation d''un redémarrage. Ne démarre aucun build.
+SafeRestartCommand.ShortDescription=Redémarre Jenkins en sûreté. Ne démarre aucun build.
+CancelQuietDownCommand.ShortDescription=Annule l''effet de la commande "quiet-down".
+OfflineNodeCommand.ShortDescription=Arrête temporairement d''utiliser un nœud dédié aux builds, jusqu''à la prochaine commande "online-node".
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/cli/Messages_fr.properties:89]:
```suggestion
OfflineNodeCommand.ShortDescription=Arrête temporairement d''utiliser un nœud dédié aux builds jusqu''à la prochaine commande "online-node".
```
Comma not useful in French.

File: core/src/main/resources/hudson/diagnosis/NullIdDescriptorMonitor/message_fr.properties [hunk 9]
```diff
@@ -0,0 +1,3 @@
+blurb=Les extensions suivantes n''ont aucune valeur "ID" et peuvent en conséquence causer des problèmes. Veuillez mettre à jour ces plugins s''ils ne sont pas dans la dernière version, \
+  et s''ils sont déjà dans la dernière version, merci de remonter le problème afin que nous puissions le corriger
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/diagnosis/NullIdDescriptorMonitor/message_fr.properties:2]:
```suggestion
  et s''ils sont déjà dans la dernière version merci de remonter le problème afin que nous puissions le corriger
```
Comma not mandatory in French here, increase the link between these two parts (subjective)

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/description_fr.properties [hunk 10]
```diff
@@ -0,0 +1,2 @@
+blurb = Cet avertissement informe les administrateurs lorsque les fichiers de configuration Jenkins contiennent des données illisibles. \
+  Généralement ceci est dû à des plugins avec des descentes de version, ou ayant subi des modifications non-compatibles avec des versions précédentes.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/diagnosis/OldDataMonitor/description_fr.properties:2]:
```suggestion
  Généralement ceci est dû à des plugins avec des descentes de version, ou ayant subi des modifications incompatibles avec des versions précédentes.
```

File: core/src/main/resources/hudson/slaves/SlaveComputer/threadDump_fr.properties [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNEC
+
+title={0} Thread Dump
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/slaves/SlaveComputer/threadDump_fr.properties:22]:
```suggestion
title={0} Dump de thread
```
🤷 😛 (optional)

File: core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/message_fr.properties (hunk 7 above)
[the remark this answers is not in the document]
Comment (jtnord) [reply core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/message_fr.properties:3]: https://www.lr-coordination.eu/node/251

**Review by jtnord — APPROVED** [review]
could do with a review from someone that speaks French better than me :)

File: core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/description_fr.properties [hunk 12]
```diff
@@ -0,0 +1 @@
+blurb = Ce message avertit les administrateurs que les déclencheurs ne se déclenchent pas comme planifié, typiquement car des plugins exécutent des actions additionnelles (comme des accès réseau) durant l''exécution des déclencheurs.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/description_fr.properties:1]:
```suggestion
blurb = Ce message avertit les administrateurs que les déclencheurs ne suivent pas le planning, typiquement car des plugins exécutent des actions additionnelles (comme des accès réseau) durant l''exécution des déclencheurs.
```
Not an easy one ^^ Optional suggestion.

File: core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message_fr.properties [hunk 13]
```diff
@@ -0,0 +1,3 @@
+blurb = Les déclencheurs suivants ont été lents à s''exécuter. \
+  Ceci est généralement dû à des problèmes réseau ou des charges hautes sur le contrôleur Jenkins. \
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/triggers/SlowTriggerAdminMonitor/message_fr.properties:2]:
```suggestion
  Ceci est généralement dû à des problèmes réseau ou des charges importantes sur le contrôleur Jenkins. \
```
Subjective

File: core/src/main/resources/hudson/util/IncompatibleVMDetected/index_fr.properties [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+errorMessage=\
+        Nous avons détecté que votre JVM n''est pas supportée par Jenkins. \
+        Ceci est dû à des limitations d''une librairie utilisée par Jenkins, notamment XStream. \
+        Consulter <a href="https://x-stream.github.io/faq.html">this FAQ</a> pour plus de détails.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/util/IncompatibleVMDetected/index_fr.properties:26]:
```suggestion
        Consulter <a href="https://x-stream.github.io/faq.html">cette FAQ</a> pour plus de détails.
```

File: core/src/main/resources/hudson/util/InsufficientPermissionDetected/index_fr.properties [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#  THE SOFTWARE.
+
+errorMessage.1=\
+        Nous avons détecté que Jenkins ne possède pas suffisamment de droits pour s''exécuter, \
+        comme indiqué par la stack trace ci-dessous. La cause de ce problème est certainement \
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/util/InsufficientPermissionDetected/index_fr.properties:25]:
```suggestion
        comme indiqué par la trace d''appels ci-dessous. La cause de ce problème est certainement \
```
Optional

File: core/src/main/resources/hudson/util/InsufficientPermissionDetected/index_fr.properties [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+        dûe à l''activation du gestionnaire de sécurité. Si cela était voulu, \
+        vous devez autoriser des droits suffisants pour exécuter Jenkins. \
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/util/InsufficientPermissionDetected/index_fr.properties:27]:
```suggestion
        vous devez donner des droits suffisants à Jenkins pour son exécution. \
```

File: core/src/main/resources/hudson/util/InsufficientPermissionDetected/index_fr.properties [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+        Si ce n''était pas le cas ou si vous ne savez pas ce qu''est un gestionnaire de sécurité, alors la \
+        solution la plus facile pour corriger le problème est simplement de désactiver le gestionnaire de sécurité.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/util/InsufficientPermissionDetected/index_fr.properties:29]:
```suggestion
        solution la plus facile pour corriger le problème est de simplement désactiver le gestionnaire de sécurité.
```
Subjective

File: core/src/main/resources/hudson/util/JenkinsReloadFailed/index_fr.properties [hunk 18]
```diff
@@ -0,0 +1 @@
+msg=Jenkins a été interrompu pendant qu''il rechargeait des données depuis le disque. Pour éviter des pertes non prévues de données, Jenkins a arrêté de fonctionner. Veuillez redémarrer Jenkins.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/util/JenkinsReloadFailed/index_fr.properties:1]:
```suggestion
msg=Jenkins a été interrompu pendant qu''il rechargeait des données depuis le disque. Pour éviter des pertes de données non prévues, Jenkins a arrêté de fonctionner. Veuillez redémarrer Jenkins.
```
Optional

File: core/src/main/resources/hudson/util/NoHomeDir/index_fr.properties [hunk 19]
```diff
@@ -0,0 +1,30 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+errorMessage.1=\
+   Impossible de créer le répertoire home ‘{0}’. Cela est certainement dû à un problème de droits.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/util/NoHomeDir/index_fr.properties:24]:
```suggestion
   Impossible de créer le répertoire "home" ‘{0}’. Cela est certainement dû à un problème de droits.
```
Optional

File: core/src/main/resources/hudson/util/NoHomeDir/index_fr.properties [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+errorMessage.2=\
+   Afin de changer le répertoire home, utilisez la variable d''environnement <code>JENKINS_HOME</code> ou déclarez \
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/util/NoHomeDir/index_fr.properties:26]:
```suggestion
   Afin de changer le répertoire "home", utilisez la variable d''environnement <code>JENKINS_HOME</code> ou déclarez \
```

File: core/src/main/resources/hudson/views/BuildButtonColumn/column_fr.properties [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+Task_scheduled={0} planifié
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/views/BuildButtonColumn/column_fr.properties:23]:
```suggestion
Task_scheduled={0} planifiée
```
If it's a task

File: core/src/main/resources/hudson/views/BuildButtonColumn/column_fr.properties [hunk 22]
```diff
@@ -0,0 +1,25 @@
...
+Schedule_a_task=Planifie un {1} pour {0}
+Schedule_a_task_with_parameters=Planifie un {1} avec les paramètres de build {0}
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/views/BuildButtonColumn/column_fr.properties:25]:
```suggestion
Schedule_a_task=Planifie une {1} pour {0}
Schedule_a_task_with_parameters=Planifie une {1} avec les paramètres de build {0}
```
Same as above, not sure if applicable

**Review by lemeurherve — COMMENTED** [review]
Another great PR, 👏

**Comment by lemeurherve** [comment]
Nit: there is a typo in the title, JENKINS is missing its "i" 😉

File: core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/description_fr.properties (hunk 1 above)
Comment (jgreffe) [reply core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/description_fr.properties:2]: Just a typo in suggestion: `est fournie`

Comment (lemeurherve) [reply core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/description_fr.properties:2]: D'oh, good catch! I've edited my suggestion 🙂

File: core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/description_fr.properties [hunk 23]
```diff
@@ -0,0 +1 @@
+blurb = Quand Jenkins détecte un cycle de dépendences entre des plugins (un problème dans un ou plusieurs plugins), ce message est affiché.
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/description_fr.properties:1]:
```suggestion
blurb = Quand Jenkins détecte un cycle de dépendances entre plugins (un problème dans un ou plusieurs plugins), ce message est affiché.
```

**Review by lemeurherve — COMMENTED** [review]
Missed that one, looks good otherwise (except the PR title, still missing that "i" and still triggering me 😅)

**Review by NotMyFault — APPROVED** [review]
Oui?

File: core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/description_fr.properties (hunk 23 above)
Comment (lemeurherve) [reply core/src/main/resources/hudson/PluginManager/PluginCycleDependenciesMonitor/description_fr.properties:1]: "dépendances" is a required fix, the deletion of "des" is a nice to have.

**Comment by jgreffe** [comment]
> Missed that one, looks good otherwise (except the PR title, still missing that "i" and still triggering me 😅)

Updated :) 🔍

**Review by lemeurherve — APPROVED** [review]
LGTM!

**Pull request merged**